t_reconf/test_reconf_base.py

```diff
@@ -272,6 +273,62 @@ def test_reconf(self, name, first_config, second_config, expected_response_on_44
         client.wait_for_response(strict=expected_response_on_444_port, timeout=1)
 
 
+class TestListenStartFail(tester.TempestaTest):
+    backends = [
+        {
+            "id": "deproxy",
+            "type": "deproxy",
+            "port": "8000",
+            "response": "static",
+            "response_content": "HTTP/1.1 200 OK\r\nContent-Length: 0\r\n\r\n",
+        }
+    ]
+
+    stop = False
+
+    @classmethod
+    def setUpClass(cls):
+        cert_path = f"{TEMPESTA_WORKDIR}/tempesta.crt"
+        key_path = f"{TEMPESTA_WORKDIR}/tempesta.key"
+        cgen = CertGenerator(cert_path, key_path, True)
+        remote.tempesta.copy_file(cert_path, cgen.serialize_cert().decode())
+        remote.tempesta.copy_file(key_path, cgen.serialize_priv_key().decode())
+        super().setUpClass()
+
+    def __heavy_load(self):
+        while not self.stop:
+            with self.assertRaises(error.ProcessBadExitStatusException):
+                remote.tempesta.run_cmd("curl -i -k 127.0.0.1:443")
+
+    def test_start_failed_under_heavy_load(self):
```

Comment:
Please add docstring for the test. it does not contain clear checks and it's not clear why we need this test.